docker-compose.v3preview.yml

```diff
@@ -42,6 +42,7 @@ services:
       LANGFUSE_S3_MEDIA_UPLOAD_ENDPOINT: ${LANGFUSE_S3_MEDIA_UPLOAD_ENDPOINT:-http://minio:9000}
       LANGFUSE_S3_MEDIA_UPLOAD_FORCE_PATH_STYLE: ${LANGFUSE_S3_MEDIA_UPLOAD_FORCE_PATH_STYLE:-true}
       LANGFUSE_S3_MEDIA_UPLOAD_PREFIX: ${LANGFUSE_S3_MEDIA_UPLOAD_PREFIX:-media/}
+      LANGFUSE_SDK_ADMIN_ENABLED: ${LANGFUSE_SDK_ADMIN_ENABLED:-true}
```

Comment:
I think the naming is confusing here.

Reply:
changed it to `LANGFUSE_JS_SDK_LOCAL_EVENT_EXPORT_ENABLED`